spec/app_spec.rb

```diff
@@ -0,0 +1,145 @@
+require_relative '../app'
+require 'item_repository'
+require 'order_repository'
+
+RSpec.describe Application do
+  def reset_app_table
+    seed_sql = File.read('spec/orders_seeds.sql')
+    connection = PG.connect({ host: '127.0.0.1', dbname: 'shop_manager_test' })
+    connection.exec(seed_sql)
+  end
+
+  before(:each) do 
+    reset_app_table
+  end
+
+  it "print the header and " do
```

Comment:
Is this test name accurate? It looks like it tests the outcome of somebody not entering a number?